pyproject.toml

```diff
@@ -22,7 +22,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.6.1"
-click = "^8.0"
+click = "^7.1"
```

Comment:
Might even consider explicitly allowing both, i.e. `>=7.1, <9.0`) so as to avoid encountering this issue in reverse if/when Celery moves to using 8.x?